browser/app/js/buckets/__tests__/actions.test.js

```diff
@@ -174,7 +171,6 @@ describe("Buckets actions", () => {
     const expectedActions = [
       { type: "alert/SET", alert: {id: 0, message: "Bucket 'test3' has been deleted.", type: "info"} },
       { type: "buckets/REMOVE", bucket: "test3" },
-      { type: "buckets/SET_LIST", buckets: ["test1", "test2"] },
```

Comment:
why is this test removed?

Reply:
These tests shouldn't be removed @BigUstad